stackdriver.go

```diff
@@ -190,9 +196,14 @@ type Options struct {
 	// trace and metric clients, and then every time a new batch of traces or
 	// stats needs to be uploaded.
 	//
+	// Do not set a timeout on this context. Instead, set the Timeout option.
+	//
 	// If unset, context.Background() will be used.
 	Context context.Context
 
+	// Timeout for all API calls. If not set, defaults to 2 seconds.
```

Comment:
Why the default of 2 seconds? That seems like a really short time to me and moreover for slow connections or network saturated connections this seems like we are just going to drop spans.

Reply:
Will increase to 5 seconds. I think dropping spans is preferable to causing a memory issue by having too long a timeout.